art/art.py

```diff
@@ -382,6 +382,9 @@ def indirect_decoration(decoration):
     :return: decoration as str
     """
     decorations = DECORATION_NAMES
+    if decoration == "random" or decoration == "rand" or decoration == "rnd":
```

Comment:
what about `decoration in ["random", "rand", "rnd"]`?

Reply:
You are right!
But I have a plan to solve this issue + some other minor issues in a separate PR (immediately after #138)